spec/controllers/graphql/queries/freaks_spec.rb

```diff
@@ -13,29 +13,14 @@ module Graphql
     end
 
     before do
-      create(:freak, first_name: 'Ion')
-      create(:freak, first_name: 'Gheorghe')
+      role = create(:role, name: 'Software developer')
+      create(:freak, role_id: role.id)
```

Comment:
You can use FactoryBot traits to improve this.

But looking over the freak factory, it seems to have an association role, so is it really necessary to create a new role here since the freak already has one? o.o